src/l200geom/cli.py

```diff
@@ -42,6 +42,38 @@ def dump_gdml_cli(argv: list[str] | None = None) -> None:
         public_geometry=args.public_geom,
     )
 
+    if args.print_volumes:
+        try:
+            if args.print_volumes == "logical":
+                lines = []
+                for name, lv in registry.logicalVolumeDict.items():
+                    solid = getattr(lv, "solid", None)
+                    solid_type = solid.__class__.__name__ if solid is not None else "UnknownSolid"
+                    material = getattr(lv, "material", None)
+                    mat_name = getattr(material, "name", "UnknownMaterial")
+                    density = getattr(material, "density", "UnknownDensity")
+                    lines.append(f"{name} is a {solid_type} made of {mat_name} of density {density} g/cm3")
+            else:
+                lines = []
+                for name, pv in registry.physicalVolumeDict.items():
+                    copy_nr = getattr(pv, "copyNumber", None)
+                    if copy_nr is None:
+                        copy_nr = getattr(pv, "copy", None)
+                    if copy_nr is None:
+                        copy_nr = getattr(pv, "copy_no", "?")
+                    lv = getattr(pv, "logicalVolume", None)
+                    lv_name = getattr(lv, "name", None) if lv is not None else None
+                    if lv_name is None:
+                        lv_name = getattr(pv, "logicalVolumeName", "?")
+                    lines.append(f"{name} ({copy_nr}) built from logical: {lv_name}")
+            for line in sorted(lines):
+                print(line)  # noqa: T201
+        except AttributeError:
+            log.error(
```

Comment:
why is this error handling in? Is there any relevant case in which this can happen?